server/src/sim.rs

```diff
@@ -272,6 +273,14 @@ impl Sim {
         let span = error_span!("step", step = self.step);
         let _guard = span.enter();
 
+        {
+            let mut ticker_query = self.world.query::<&mut TickerEntity>();
+            let ticker_iter = ticker_query.iter();
+            for (_entity, ticker_entity) in ticker_iter {
```

Comment:
You can shorten this to
```rust
for (_entity, ticker_entity) in self.world.query::<&mut TickerEntity>().iter() {
```
in which case you'll also be able to get rid of the braces, since the scope will already be limited to the `for` loop.